### PR TITLE
Added info from Gautham

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/servers-create.adoc [hunk 1]
```diff
@@ -42,16 +42,20 @@ The `amc_setup` script calls the Runtime Manager Agent installation script, whic
...
+=== IBM JVM
 
-When creating a server using IBM's Java Virtual Machine (JVM) you must use a different truststore than the default truststore installed by the Runtime Manager Agent.
+When creating a server using the IBM Java Virtual Machine (JVM), you must use a different truststore than the default truststore installed by the Runtime Manager Agent.
 
-. Download the custom truststore from the customer support portal.
-. Rename the file to `truststore.jks`.
+. Download the custom truststore from this knowledge article:
++
+https://support.mulesoft.com/s/article/Connectivity-issues-between-Mule-Runtime-and-Anypoint-Runtime-manager-using-an-IBM-JVM[Connectivity issues between Mule Runtime and Anypoint Runtime Manager using an IBM JVM]
```
Comment (valkyrie69) [inline modules/ROOT/pages/servers-create.adoc]: For versions, our convention is earlier and later. Use version blah blah or later

File: modules/ROOT/pages/servers-create.adoc [hunk 2]
```diff
@@ -42,16 +42,20 @@ The `amc_setup` script calls the Runtime Manager Agent installation script, whic
...
+. Rename the file:
+* `anypoint-truststore.jks` — Mule Runtime 4.1.3 (Mule Agent 2.1.4) and higher
+* `truststore.jks` — Mule Runtime 4.1.2 (Mule Agent 2.1.3)
 . Copy the custom truststore into the Runtime Manager Agent `<MULE_HOME>/conf` folder.
 . Restart the Mule runtime.
```
Comment (valkyrie69) [inline modules/ROOT/pages/servers-create.adoc]: This was already here but we aren't supposed to use Mule runtime anymore. It has to be Mule runtime engine or just Mule.

**Review by valkyrie69 — APPROVED** [review]
Minor comments

File: modules/ROOT/pages/servers-create.adoc [hunk 3]
```diff
@@ -36,22 +36,26 @@ The `amc_setup` command is located in the `$MULE_HOME/bin` directory. If you do
...
+When creating a server using the IBM Java Virtual Machine (JVM), you must use a different truststore than the default truststore installed by the Runtime Manager agent.
 
-. Download the custom truststore from the customer support portal.
-. Rename the file to `truststore.jks`.
-. Copy the custom truststore into the Runtime Manager Agent `<MULE_HOME>/conf` folder.
-. Restart the Mule runtime.
+. Download the custom truststore from this knowledge article:
++
+https://support.mulesoft.com/s/article/Connectivity-issues-between-Mule-Runtime-and-Anypoint-Runtime-manager-using-an-IBM-JVM[Connectivity issues between Mule and Anypoint Runtime Manager using an IBM JVM]
+. Rename the file:
```
Comment (raogautham) [inline modules/ROOT/pages/servers-create.adoc]: "Rename the truststore file under $MULE_HOME/conf folder to:"

**Review by raogautham — APPROVED** [review]

**Pull request merged**